### PR TITLE
[ISSUE #1910] fix the default_group of nacos.

### LINKED ISSUE #1910: the default group of nacos is lowercase
## Describe the bug

The problem arises in two places, located in：

- NacosServerRegisterRepository
- NacosClientRegisterRepository

image

image

nacos's group name is case sensitive.  

image

the default_group of naocs  is defined in  `shenyu-common/src/main/java/org/apache/shenyu/common/constant/NacosPathConstants.java` ， we can use it 

## Environment

- ShenYu version(s): 2.4.0
- Nacos version(s): 2.0.2

## Steps to reproduce

start shenyu-examples-springcloud

## Debug logs

<details>
<summary>Click to expand</summary>

```
replace this line with your debug logs
```
</details>

### PR DESCRIPTION
**Pull request by lltime**
// Describe your PR here; eg. Fixes #1910 

Make sure that:

- [x] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor/).
- [ ] You submit test cases (unit or integration tests) that back your changes.
- [ ] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: shenyu-register-center/shenyu-register-client/shenyu-register-client-nacos/src/main/java/org/apache/shenyu/register/client/nacos/NacosClientRegisterRepository.java [hunk 1]
```diff
@@ -47,7 +48,7 @@
 @Slf4j
 public class NacosClientRegisterRepository implements ShenyuClientRegisterRepository {
 
-    private String defaultGroup = "default_group";
+    private final String defaultGroup = NacosPathConstants.GROUP;
```
Comment (KevinClair) [inline shenyu-register-center/shenyu-register-client/shenyu-register-client-nacos/src/main/java/org/apache/shenyu/register/client/nacos/NacosClientRegisterRepository.java:52]: No need to create another variable. just use `NacosPathConstants.GROUP`.

**Comment by SaberSola** [comment]
please check the ci

**Comment by lltime** [comment]
> please check the ci

This is my first time to submit the code, can you teach me how to check ？ Thx

**Comment by SaberSola** [comment]
This branch has conflicts that must be resolved. please resolved it

**Comment by lw1243925457** [comment]
> > please check the ci
> 
> This is my first time to submit the code, can you teach me how to check ？ Thx

pull the lastest code, resolve the conficts.

**Comment by midnight2104** [comment]
hi, @lltime , you can resolve conflict following as:
```bash
git checkout master

git fetch upstream && git reset --hard upstream/master

git checkout #yourBranch

git merger matser

// resolve conflict

git commit #comment

git push origin #yourBranch
```

if you have any question, pls contact me

**Review by lw1243925457 — APPROVED** [review]

**Comment by lw1243925457** [comment]
It is good to change the variable to a constant.
This kind of conflict caused by backward codeI，not good for novices，so I resolved it.

**Pull request merged**